src/components/Card/Card.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render } from '@testing-library/react-native';
+import '@testing-library/jest-native/extend-expect';
+import { ThemeProvider } from 'styled-components/native';
+import { Text } from 'react-native';
+import theme from '../../common/themeSelectors/theme/mock-theme.json';
+import { Card, CardTypes } from './Card';
+
+jest.useFakeTimers();
+
+const renderCard = (fn, props?: CardTypes) => fn(
+    <ThemeProvider theme={theme}>
+      <Card>
+        <Text>{props}</Text>
+      </Card>
+    </ThemeProvider>,
+);
+
+describe('Card component', () => {
+  it('Should render Base type as default', () => {
```

Comment:
Sugestão: aqui, o `should` pode ficar minúsculo. É um detalhezinho, mas na hora do relatório de teste, ele aparecerá como uma frase: `Card component should render Base type as default`.